content/reference/endpoints/rental_cancelation_policies.md

```diff
@@ -3,6 +3,12 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents Rental Cancelation Policies which define the rules and potential penalties for the cancelation of [Bookings](/reference/endpoints/bookings/) (defined either via Rental Cancelation Policy Items or `preset`)
```

Comment:
Just to avoid potential confusion I would use `describe` instead of `define`. It's cancelation policy items that define the penalty behaviour, cancelation policy itself is just an informative text without specific contribution to the penalty.
```suggestion
This resource represents Rental Cancelation Policies which describe the rules and potential penalties for the cancelation of [Bookings](/reference/endpoints/bookings/) (defined either via Rental Cancelation Policy Items or `preset`)
```